libpyEM/EMAN2_utils.py

```diff
@@ -313,10 +313,6 @@ def makepath(options, stem='e2dir'):
 	
 	return options
 
-#### this triggers a segmentation fault. in case we need it somewhere...
-def do_segfault():
-	exec'()'*7**6
```

Comment:
@g5v991x , I am not sure how to fix this, so I removed it. If you want to, you are welcome to fix it and add it back.